admin_manual/configuration_files/files_locking_enabling.rst

```diff
@@ -16,4 +17,28 @@ underlying filesystem.
 The Files Locking app has no configuration options; all you need to do is 
 enable or disable it on your Apps page.
 
-.. figure:: ../images/files_locking_app.png
\ No newline at end of file
+.. figure:: ../images/files_locking_app.png
+
+We recommend using `Redis <http://redis.io/>`_ as your ownCloud memcache when 
+you enable the Files Locking app. Memcached, the popular distributed memory 
```

Comment:
> Really? I thought that files_locking is for locking against concurrent processes to the ownCloud server on the server side (filesystem level).

There is no way to lock files in PHP reliably against other processes. :see_no_evil: – PHP uses advisory based locking per default, see http://php.net/manual/en/function.flock.php and especially enjoy:

> flock() uses mandatory locking instead of advisory locking on Windows. Mandatory locking is also supported on Linux and System V based operating systems via the usual mechanism supported by the fcntl() system call: that is, if the file in question has the setgid permission bit set and the group execution bit cleared. On Linux, the file system will also need to be mounted with the mand option for this to work.